nix/third-party/gRPC-haskell/core/chs/Network/GRPC/Unsafe.hs

```diff
@@ -320,6 +320,11 @@ grpcShutdown =
   grpcShutdown'_ >>
   return ()
 
+grpcShutdownBlocking :: IO ()
+grpcShutdownBlocking =
```

Comment:
`void grpcShutdownBlocking'_`?

Reply:
This should be autogenerated when `c2hs` works properly so I kept it consistent with what `c2hs` generates. The only reason we have this source file is that `c2hs` on Windows fails in weird ways.

Reply:
Makes sense. Thanks for the explanation.